### PR TITLE
Add 'poly' learning rate policy

### PR DESCRIPTION
**Pull request by bes-dev**
Added implementation one of the most popular learning rate policy for semantic segmentation.
It was been used in:
PSPNet: https://arxiv.org/abs/1606.00915
DeepLab: https://arxiv.org/abs/1612.01105
and in many other architectures.

### CODE REVIEW

**Comment by ezyang** [comment]
Ignore the CI failures.

This needs a test.

**Comment by BelBES** [comment]
@ezyang I was add a test for PolyLR

**Review by ezyang — CHANGES_REQUESTED** [review]
You have Python syntax errors. Fixing lint will be a good start to fix: https://travis-ci.org/pytorch/pytorch/jobs/440758812

**Comment by BelBES** [comment]
@ezyang issue was fixed. Please look at PR.

**Comment by ezyang** [comment]
@BelBES In your PR you refer to `max_iterations` but this is not set anywhere. What did you mean to refer to here? I'm going to close this PR for now, feel free to reopen if you know how to fix this problem.
